### PR TITLE
chore: remove more unused exports

### PR DESCRIPTION
**Pull request by vm**
Removing even more unused exports

### CODE REVIEW

File: src/components/Button/index.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { RowBetween } from '../Row'
 
 type ButtonProps = Omit<ButtonPropsOriginal, 'css'>
 
-export const BaseButton = styled(RebassButton)<
+const BaseButton = styled(RebassButton)<
```
Comment (grabbou) [inline src/components/Button/index.tsx:10]: this is awesome, impossible to spot without a linter, can be confusing to see exported members while refactoring

File: src/components/Menu/index.tsx [hunk 2]
```diff
@@ -146,179 +90,6 @@ const InternalMenuItem = styled(Link)`
...
-                )
-            }
-          })()}
-      </StyledMenu>
-      <PrivacyPolicyModal />
-      <FeatureFlagModal />
-    </>
-  )
-}
-
```
Comment (grabbou) [inline src/components/Menu/index.tsx:321]: Looks like this file exports another member `NewMenu` (can't see it in the diff, have to check the file). Let's rename that to `export default` now that we got rid of the `OldMenu`

File: src/components/Popups/SurveyPopup.tsx [hunk 3]
```diff
@@ -51,7 +51,7 @@ const WrappedCloseIcon = styled(X)`
 
 const END_TIMESTAMP = 1642272346 // Jan 15th
 
-export default function SurveyPopup() {
+function SurveyPopup() {
```
Comment (vm) [inline src/components/Popups/SurveyPopup.tsx:54]: this entire file is unused it seems

File: src/components/Popups/SurveyPopup.tsx [hunk 4]
```diff
@@ -51,7 +51,7 @@ const WrappedCloseIcon = styled(X)`
...
   const theme = useTheme()
```
Comment (grabbou) [inline src/components/Popups/SurveyPopup.tsx:55]: How did that pass your local linter 🫣

Comment (grabbou) [reply src/components/Popups/SurveyPopup.tsx:55]: This is really interesting, we now have a file that has no exported members... I would assume it should be automatically removed!

**Review by grabbou — CHANGES_REQUESTED** [review]
LGTM, except for one lint issue and one nit 🤞

File: src/components/Popups/SurveyPopup.tsx (hunk 4 above)
Comment (vm) [reply src/components/Popups/SurveyPopup.tsx:55]: it's because the other thing is just a warning, not an error!

**Review by yyip-dev — APPROVED** [review]
Lgtm

**Pull request merged**